lib/membrane_rtmp_plugin/rtmp/source/message_validator.ex

```diff
@@ -1,44 +1,30 @@
-defmodule Membrane.RTMP.MessageValidator do
+defprotocol Membrane.RTMP.MessageValidator do
```

Comment:
Is there any benefit to use the protocol instead of a simple behaviour? Using a protocol only adds some boilerplate without any other benefit (but I'm opened to conversation).

Reply:
Also with a behaviour we are able to provide default implementations for certain callbacks, with protocols it is no longer possible.